src/main/entity/Promotion.ts

```diff
@@ -168,6 +170,13 @@ export class Promotion {
   @Column()
   restaurantName: string;
 
+  /*
+   * Represents restaurant location from Google Places API
+   * Many promotions can come from the same restaurant and thus have the same restaurantLocation
```

Comment:
We're not allowed to store this though right?

Reply:
We'll be storing the user inputted version! We need this in order to refresh the invalid place ID on the FE

Reply:
Ohya but the comment says its from Google places api, could we update that?

Reply:
Ooh I kept it as "Google Places API" cause last time I changed `restaurantName`'s description to something along "inputted by user"- you mentioned that it was giving away too much FE implementation details?

Reply:
Hmm not sure where I implied that, this point in ur previous PR was for discussion https://github.com/ubclaunchpad/PromoPal-backend/pull/81#discussion_r551717463. If I did, sorry that was my mistake.

I realized we can't store a restaurants name or address from google places API. So then `restaurantName` and `restaurantAddress` don't represent anything from google places API

Reply:
How about 
`The restaurant address of the promotion`

Reply:
Hmmm, that wasn't the comment I was referring to-I believe you made the point in chat instead-but sure, I can change it to that

Reply:
My bad, but does it make sense how saying google places API doesn't really represent what this property is?

Reply:
Yup! I thought it was weird when we kept it as that last time so sounds good to me!